app/src/main/java/org/simple/clinic/scanid/ScanBpPassportActivity.kt

```diff
@@ -34,14 +33,21 @@ class ScanBpPassportActivity: AppCompatActivity(), ScanSimpleIdScreen.ScanResult
   lateinit var locale: Locale
 
   @Inject
-  lateinit var component: ScanBpPassportActivityComponent
+  lateinit var features: Features
 
   @Inject
-  lateinit var features: Features
+  lateinit var component: ScanBpPassportActivityComponent
+
+  private lateinit var binding: ScreenScanSimpleBinding
+
+  private val scanBpPassportView
+    get() = binding.scanBpPassportView
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.screen_scan_simple)
+
+    binding = ScreenScanSimpleBinding.inflate(layoutInflater)
```

Comment:
We need to clear the binding in accordance with the view lifecycle.

Reply:
Same comment as above. We don't need to do that for Activities.